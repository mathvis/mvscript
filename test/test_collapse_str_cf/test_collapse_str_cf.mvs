if ("abcabc" > "abc") {
	let a = 0;
}
