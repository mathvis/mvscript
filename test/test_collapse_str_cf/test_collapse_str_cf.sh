#!/bin/bash
helix test_collapse_str_cf.mvs test_collapse_str_cf.out --vsplit
