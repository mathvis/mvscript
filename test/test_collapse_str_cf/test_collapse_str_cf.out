Decl (CollapsedControlFlow (Block If [Decl (Variable (VarIdentifier "a") (Just int) (Just (Type (Int 0))))]))
